plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraColumnHandle.java

```diff
@@ -44,7 +45,7 @@ public void testRoundTrip2()
         CassandraColumnHandle expected = new CassandraColumnHandle(
                 "name2",
                 1,
-                CassandraType.MAP,
+                DataType.Name.MAP,
```

Comment:
It feels to me that we should keep `CassandraType` here. `DataType.Name` does not pass all the needed information if we want to support parametrized types.